app/views/posts/index.html.erb

```diff
@@ -23,14 +23,17 @@
       <div class='comment-cont'>
         <% post.comment_5_recent.each do |comment| %>
         <p> <%="#{comment.author.name}: #{comment.text} " %></p>
-
         <%end%>
-      
-      </div>
-      
-        
+      </div>       
     </div>
+    
 
   <%end%>
+  <div>
+    <button>
```

Comment:
- [ ] I am seeing these details(attached a screenshot) before the `Add a post` button in the user's show and posts index page. Please fix this.

![Screenshot from 2022-02-24 02-06-31](https://user-images.githubusercontent.com/22434660/155405159-2525c91b-7290-47c6-a48b-98418da9d4b9.png)